ftrackhq/ftrack-recipes
# Add per project location example

Resolves : 

* CLICKUP-3k0q42b
* ZENDESK-97919

This PR has been tested on :



## Changes

This is a new example custom location to make use of the legacy settings per project to allow users to define custom root folder per projects.

the code provides a new custom Location and a resolver to bypass some core limitation of the original implementation.

## Test

* Install the plugin
* set the project's Storage / project folder to a custom folder.
* publish something with connect.
    
the component should reflect the custom project folder defined in each different project.
